documentation/docs/mock-apps/services/process-object-event.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 26
+title: Process Object Event
+---
+
+import Disclaimer from '../../\_disclaimer.mdx';
+
+<Disclaimer />
+
+## Description
+
+The `processObjectEvent` service is responsible for processing an [Object Event (DTE)](https://uncefact.github.io/spec-untp/docs/specification/DigitalTraceabilityEvents), issuing a [Verifiable Credential (VC)](https://uncefact.github.io/spec-untp/docs/specification/VerifiableCredentials), uploading it to the [Storage service](/docs/mock-apps/dependent-services/storage-service), registering the link to the stored DTE with the [Identity Resolver service](/docs/mock-apps/dependent-services/identity-resolution-service). It handles the entire lifecycle of creating and managing an object event, from data input to storage and resolution.
+
+## Diagram
+
+```mermaid
+sequenceDiagram
+participant C as Client
+participant P as processObjectEvent
+participant V as VCKit
+participant S as Storage
+participant D as DLR
+C->>P: Call processObjectEvent(objectEvent, context)
+P->>P: Validate context
+P->>P: Extract identifier
+P->>V: Issue VC
+V-->>P: Return VC
+P->>S: Upload VC
+S-->>P: Return VC URL
+P->>D: Register link resolver
+D-->>P: Return resolver URL
+P-->>C: Return object event VC and resolver URL
+```
+
+## Example
+
+```json
+{
+  "name": "processObjectEvent",
+  "parameters": [
+    {
+      "vckit": {
+        "vckitAPIUrl": "https://api.vckit.example.com",
+        "issuer": "did:example:123456789abcdefghi"
+      },
+      "epcisObjectEvent": {
+        "context": ["https://www.w3.org/2018/credentials/v1", "https://gs1.org/voc/"],
+        "type": ["VerifiableCredential", "ObjectEventCredential"],
+        "renderTemplate": [
+          {
+            "template": "<div><h2>Object Event</h2></div>",
+            "@type": "WebRenderingTemplate2022"
+          }
+        ],
+        "dlrIdentificationKeyType": "gtin",
+        "dlrLinkTitle": "Object Event",
+        "dlrVerificationPage": "https://verify.example.com"
+      },
+      "storage": {
+        "url": "https://storage.example.com/upload",
+        "params": {
+          "resultPath": "/url"
+        }
```

Comment:
I noticed you've used a `bucket` property in the `params` object within the `processObjectEvent` services config in the `app-config.json` file. Should we include this in the documentation to make it clearer?

Reply:
Good point! I will add it